src/guides/v2.4/graphql/mutations/copy-items-between-requisition-list.md

```diff
@@ -0,0 +1,90 @@
+---
+group: graphql
+title: copyItemsBetweenRequisitionList mutation
+b2b_only: true
+contributor_name: Zilker Technology
+contributor_link: https://www.ztech.io/
+---
+The copyItemsBetweenRequisitionList mutation copies items from one requisition list to another.
```

Comment:
Wrap the mutation name with backticks. Each topic contains multiple un-highlighted mutation names.